external/storm-solr/src/main/java/org/apache/storm/solr/bolt/AbstractSolrBolt.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.storm.solr.bolt;
```

Comment:
I deleted this class as I was able to abstract the specific implementations in the mapper, which returns the SolrRequest object